### DISCUSSION TITLE
I didn't see a "set priority" cmd line argument

### DISCUSSION THREAD

**Discussion by jkr4m3r** [kangyu-california/PersistentWindows#206] [Ideas]
Doing it manually changes back to lower priority on reboot.

I found it works really great running at normal priority, troublesome at default lower priority. 

1. To make it automatic I had to export the scheduled task as xml, edit the priority to `<Priority>5</Priority>`
2. delete the original
3. Then import the edited xml and reboot.

Hope this helps.

**Comment by kangyu-california** [comment]
I found another related article on 
[How To Change Task Priority Through PowerShell](https://blog.realhe.ro/how-to-change-priority-on-windows-scheduled-task-using-powershell/)
You will need to run those cmdlet in PowerShell with admin privilege
